Examples/Python/quilt_movie.py

```diff
@@ -19,8 +19,10 @@
 
 import vtk
 
+from vtk import vtkRenderingLookingGlass
+
 # Create a looking glass render window
-renWin = vtk.vtkLookingGlassInterface.CreateLookingGlassRenderWindow()
+renWin = vtkRenderingLookingGlass.vtkLookingGlassInterface.CreateLookingGlassRenderWindow()
```

Comment:
We have to import `vtkRenderingLookingGlass` explicitly from VTK for the wheels (because it is not included in `vtkmodules/all.py`)